src/controllers/resources/list.js

```diff
@@ -82,37 +82,53 @@ export default function () {
     })
 
     if (resourceSlug === 'orders') {
-      const getAmount = statusList => {
+      const getOrders = statusList => {
         let amount = 0
-        list.forEach(row => {
+        let countStatus = 0
+        list.forEach((row, i) => {
           if (statusList) {
             const status = row['financial_status/current']
             if (!status || !statusList.includes(status)) {
               return
             }
+            countStatus++
+          } else {
+            countStatus = i + 1
           }
```

Comment:
Pra quê isso aqui? :thinking: 

```suggestion
          }
```

Reply:
Não entendi a lógica...

Reply:
Porque se não tiver status, como é o caso de pedido geral, ele não entra no if do statusList e ai eu pego o último valor iterativo e somo 1 para dar o length correto

Reply:
Para aprovado tem 2 status, para cancelado tem vários, mas para pedidos totais não tem nenhum

Reply:
Sim mas pra isso você não precisa somar nada na verdade, a lista é o array `list`, o total de pedidos listados é `list.length`

Reply:
Sim,  eu tava fazendo isso antes, mas ai como fiz pros dois, pensei em fazer nesse tbm, vou mudar la

Reply:
Não é legal porque você tá somando `i + 1` a cada interação, em uma lista de 480 pedidos você faz 480 operações quando na verdade não precisava de nenhum haha